static/js/pages/applications/ApplicationDashboardPage.js

```diff
@@ -486,7 +492,7 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
                 <h2>
                   {isNovoEdEnrolled(application) ? (
```

Comment:
This shouldn't happen in practice, but just in case, we should make sure the URL is not null 

`isNovoEdEnrolled(application) && novoedUrl ? (`